src/sage/plot/animate.py

```diff
@@ -506,26 +526,28 @@ def graphics_array(self, ncols=3):
 
         EXAMPLES::
 
+            sage: # needs sage.schemes
             sage: E = EllipticCurve('37a')
-            sage: v = [E.change_ring(GF(p)).plot(pointsize=30) for p in [97, 101, 103]]
+            sage: v = [E.change_ring(GF(p)).plot(pointsize=30)
+            ....:      for p in [97, 101, 103]]
             sage: a = animate(v, xmin=0, ymin=0, axes=False)
             sage: print(a)
             Animation with 3 frames
-            sage: a.show()  # optional -- ImageMagick
+            sage: a.show()                      # optional - imagemagick
```

Comment:
`ImageMagick` ?

Reply:
OK, I am normalizing in this file to `# optional -- ImageMagick` because that seemed to be the preference of the author

Reply:
Don't we use uncapitalized names for all optional doctest tags, except this one? If we allow an exception, then the result would just be an inconsistency. I mean that some will use `# optional - imagemagick` nonetheless.

Is the author strong about this?

Reply:
Changing it to the fixdoctests normal form is also fine with me.

Reply:
@dcoudert ?

Reply:
We have other exceptions. For instance we use both `gurobi` and `Gurobi`:
```
xyz:sage dcoudert$ git grep "# optional - [gG]urobi" src/sage/
src/sage/combinat/matrices/dancing_links.pyx:            sage: d.to_milp('gurobi')   # optional - gurobi sage_numerical_backends_gurobi
src/sage/combinat/matrices/dancing_links.pyx:            sage: s = d.one_solution_using_milp_solver('gurobi') # optional - gurobi sage_numerical_backends_gurobi
src/sage/combinat/matrices/dancing_links.pyx:            sage: s in solutions                                 # optional - gurobi sage_numerical_backends_gurobi
src/sage/features/mip_backends.py:            sage: Gurobi()._is_present()  # optional - gurobi
src/sage/knots/link.py:            sage: L.plot(solver='Gurobi')  # optional - Gurobi
src/sage/numerical/backends/cvxpy_backend.pyx:        sage: p = MixedIntegerLinearProgram(solver="CVXPY/Gurobi"); p.solve()          # optional - gurobi
src/sage/numerical/mip.pyx:            sage: p = MixedIntegerLinearProgram(solver="gurobi")   # optional - Gurobi
src/sage/numerical/mip.pyx:            sage: p.add_constraint(p[0] - p[2], min=1, max=4)      # optional - Gurobi
src/sage/numerical/mip.pyx:            sage: p.number_of_variables()                          # optional - Gurobi
```

When running `git grep "# optional - [[:upper:]]" src/sage/`, we see occurrences of: `# optional - SAGE_ROOT`, `# optional - CPLEX`, `# optional - Gurobi`, `# optional - Nonexistent_LP_solver`, `# optional - CHomP`, `# optional - FES `, `# optional - RSat`.

I don't think we want to force the use of the uncapitalized form for `SAGE_ROOT`. At least we should be consistent inside a file.

Reply:
All right then. I won't insist.